build-java/build.gradle

```
@@ -0,0 +1,31 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath "org.openapitools:openapi-generator-gradle-plugin:4.0.0-beta2"
+    }
+}
+
+apply plugin: 'org.openapi.generator'
+
+def clientOutputDir = "$projectDir/../generated-clients/java/java-clients".toString()
```

Do you need these two levels (java/java-clients)?
```suggestion
def clientOutputDir = "$projectDir/../generated-clients/java".toString()
```

---

I was thinking about a more flexible generated output structure:
```
java/java-retrofit-client
java/java-openapi-client
python/python3
...
```
It's a proposal, as the result is not being used yet. I can go to one single level for the moment.